org.eclipse.dawnsci.nexus/src/org/eclipse/dawnsci/nexus/NexusConstants.java

```diff
@@ -0,0 +1,143 @@
+/*-
+ * Copyright 2018 Diamond Light Source Ltd.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.dawnsci.nexus;
+
+/**
+ * Constants used to deal with NeXus files
+ */
+public class NexusConstants {
+	/**
+	 * Group attribute to indicate NeXus class
+	 */
+	public static final String CLASS = "NX_class";
+
+	/**
+	 * Dataset attribute name for units
+	 */
+	public static final String UNITS = "units";
+
+	/**
+	 * base classes
+	 */
+	public final static String APERTURE = "NXaperture";
+	public final static String ATTENUATOR = "NXattenuator";
+	public final static String BEAM = "NXbeam";
+	public final static String BEAMSTOP = "NXbeam_stop";
+	public final static String BENDINGMAGNET = "NXbending_magnet";
+	public final static String CAPILLARY = "NXcapillary";
+	public final static String CITE = "NXcite";
+	public final static String COLLECTION = "NXcollection";
+	public final static String COLLIMATOR = "NXcollimator";
+	public final static String CRYSTAL = "NXcrystal";
+	public final static String DATA = "NXdata";
+	public final static String DETECTORGROUP = "NXdetector_group";
+	public final static String DETECTORMODULE = "NXdetector_module";
+	public final static String DETECTOR = "NXdetector";
+	public final static String DISKCHOPPER = "NXdisk_chopper";
+	public final static String ENTRY = "NXentry";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)